### PR TITLE
Adds Anarcho Capitalism religion for chaplain

### PR DESCRIPTION
**Pull request by Orangerush**
I hope I did this all right and didn't fuck anything up. This is the first real thing I ever coded for anything so I did something that (I thought) seems pretty small and not very breakable. Adds anarcho capitalism to chaplain

### CODE REVIEW

**Comment by optimumtact** [comment]
Imposter, you've violated my NAP

**Comment by Orangerush** [comment]
Please feel free to bully me for being dumb if I fucked up

**Comment by kol1th** [comment]
meme pr

**Comment by kol1th** [comment]
`code/game/jobs/job/civilian_chaplain.dm:417:error: slot_right_hand: undefined var` 
nice

**Comment by Orangerush** [comment]
Just broke the NAP by not spawning them with a toy gun

**Comment by Shadowmech88** [comment]
Instead of their hand, why not equip it to their pocket or belt?

**Comment by Orangerush** [comment]
I give up, I don't know how to equip it to a pocket @Shadowmech88 (How embarrassing)

**Comment by Shadowmech88** [comment]
`H.equip_or_collect(new /obj/item/toy/gun(H), slot_l_store)`
This equips it to their left pocket.

**Comment by Orangerush** [comment]
cool thanks

**Comment by Probe1** [comment]
Was it because of my post by any chance?
image

**Comment by Orangerush** [comment]
Yes, I actually wanted to do that air vending machine idea someone posted but I realized I'm too new to coding to try something like that and went with what we're at now

File: code/game/jobs/job/civilian_chaplain.dm [hunk 1]
```diff
@@ -410,6 +410,11 @@
 				H.equip_or_collect(new /obj/item/clothing/mask/gas/mime(H), slot_wear_mask)
 				J = "..."
 				deity_name = "Silence"
+			if("Ancap", "Ancapistan", "MtGox")
+				B.name = "Bitcoin Wallet"
+				J = "Warlord" //
```
Comment (N3X15) [inline code/game/jobs/job/civilian_chaplain.dm]: ಠ_ಠ

File: code/game/jobs/job/civilian_chaplain.dm [hunk 2]
```diff
@@ -410,6 +410,11 @@
...
+				deity_name = "Murray Rothbard"
+				H.equip_or_collect(new /obj/item/toy/gun(H), slot_l_store)
```
Comment (N3X15) [inline code/game/jobs/job/civilian_chaplain.dm]: What

**Review by N3X15 — DISMISSED** [review]
Get a better job name

File: code/game/jobs/job/civilian_chaplain.dm (hunk 2 above)
Comment (PJB3005) [reply code/game/jobs/job/civilian_chaplain.dm]: Should use `put_in_hands` too.

Actually that said is `slot_l_store` hands or pockets?

File: code/game/jobs/job/civilian_chaplain.dm [hunk 3]
```diff
@@ -410,6 +410,11 @@
 				H.equip_or_collect(new /obj/item/clothing/mask/gas/mime(H), slot_wear_mask)
 				J = "..."
 				deity_name = "Silence"
+			if("Ancap", "Ancapistan", "MtGox")
```
Comment (CptWad) [inline code/game/jobs/job/civilian_chaplain.dm]: qdel(B)
B = new /obj/item/weapon/storage/bible/booze(H)

thank me later

**Comment by Orangerush** [comment]
I'll have to come back to this one day

File: code/game/jobs/job/civilian_chaplain.dm (hunk 2 above)
Comment (Exxion) [reply code/game/jobs/job/civilian_chaplain.dm]: Pockets

**Comment by kol1th** [comment]
That was a fast day

**Comment by Orangerush** [comment]
My autism refuses to let go until I fix it

**Comment by Orangerush** [comment]
SOON tm

**Comment by Kitfox88** [comment]
I was shooting heroin and reading “The Fountainhead” in the front seat of my privately owned police cruiser when a call came in. I put a quarter in the radio to activate it. It was the chief.

“Bad news, detective. We got a situation.”

“What? Is the mayor trying to ban trans fats again?”

“Worse. Somebody just stole four hundred and forty-seven million dollars’ worth of bitcoins.”

The heroin needle practically fell out of my arm. “What kind of monster would do something like that? Bitcoins are the ultimate currency: virtual, anonymous, stateless. They represent true economic freedom, not subject to arbitrary manipulation by any government. Do we have any leads?”

“Not yet. But mark my words: we’re going to figure out who did this and we’re going to take them down … provided someone pays us a fair market rate to do so.”

“Easy, chief,” I said. “Any rate the market offers is, by definition, fair.”

He laughed. “That’s why you’re the best I got, Lisowski. Now you get out there and find those bitcoins.”

“Don’t worry,” I said. “I’m on it.”

I put a quarter in the siren. Ten minutes later, I was on the scene. It was a normal office building, strangled on all sides by public sidewalks. I hopped over them and went inside.

“Home Depot™ Presents the Police!®” I said, flashing my badge and my gun and a small picture of Ron Paul. “Nobody move unless you want to!” They didn’t.

“Now, which one of you punks is going to pay me to investigate this crime?” No one spoke up.

“Come on,” I said. “Don’t you all understand that the protection of private property is the foundation of all personal liberty?”

It didn’t seem like they did.

“Seriously, guys. Without a strong economic motivator, I’m just going to stand here and not solve this case. Cash is fine, but I prefer being paid in gold bullion or autographed Penn Jillette posters.”

Nothing. These people were stonewalling me. It almost seemed like they didn’t care that a fortune in computer money invented to buy drugs was missing.

I figured I could wait them out. I lit several cigarettes indoors. A pregnant lady coughed, and I told her that secondhand smoke is a myth. Just then, a man in glasses made a break for it.

“Subway™ Eat Fresh and Freeze, Scumbag!®” I yelled.

Too late. He was already out the front door. I went after him.

“Stop right there!” I yelled as I ran. He was faster than me because I always try to avoid stepping on public sidewalks. Our country needs a private-sidewalk voucher system, but, thanks to the incestuous interplay between our corrupt federal government and the public-sidewalk lobby, it will never happen.

I was losing him. “Listen, I’ll pay you to stop!” I yelled. “What would you consider an appropriate price point for stopping? I’ll offer you a thirteenth of an ounce of gold and a gently worn ‘Bob Barr ‘08’ extra-large long-sleeved men’s T-shirt!”

He turned. In his hand was a revolver that the Constitution said he had every right to own. He fired at me and missed. I pulled my own gun, put a quarter in it, and fired back. The bullet lodged in a U.S.P.S. mailbox less than a foot from his head. I shot the mailbox again, on purpose.

“All right, all right!” the man yelled, throwing down his weapon. “I give up, cop! I confess: I took the bitcoins.”

“Why’d you do it?” I asked, as I slapped a pair of Oikos™ Greek Yogurt Presents Handcuffs® on the guy.

“Because I was afraid.”

“Afraid?”

“Afraid of an economic future free from the pernicious meddling of central bankers,” he said. “I’m a central banker.”

I wanted to coldcock the guy. Years ago, a central banker killed my partner. Instead, I shook my head.

“Let this be a message to all your central-banker friends out on the street,” I said. “No matter how many bitcoins you steal, you’ll never take away the dream of an open society based on the principles of personal and economic freedom.”

He nodded, because he knew I was right. Then he swiped his credit card to pay me for arresting him.

File: code/game/jobs/job/civilian_chaplain.dm [hunk 4]
```diff
@@ -410,6 +410,11 @@
 				H.equip_or_collect(new /obj/item/clothing/mask/gas/mime(H), slot_wear_mask)
 				J = "..."
 				deity_name = "Silence"
+			if("Ancap", "Ancapistan", "NAP")
```
Comment (CptWad) [inline code/game/jobs/job/civilian_chaplain.dm]: No really, add 
qdel(B)
B = new /obj/item/weapon/storage/bible/booze(H)
right under this line
Gives the chaplain the booze and money bible

**Comment by Orangerush** [comment]
@CptWad How bout that?

File: code/game/jobs/job/civilian_chaplain.dm [hunk 5]
```diff
@@ -410,6 +410,13 @@
...
+				qdel(B)
+				B = new /obj/item/weapon/storage/bible/booze(H)
```
Comment (Shadowmech88) [inline code/game/jobs/job/civilian_chaplain.dm:415]: You'll probably also need a `H.put_in_hands(B)` on the line below this.

**Comment by Orangerush** [comment]
How bout that @Shadowmech88

**Comment by Shadowmech88** [comment]
Looks fine.

**Comment by Orangerush** [comment]
Now SOON(TM)(R). good night autism

**Comment by CptWad** [comment]
👌

**Comment by gbasood** [comment]
syndicalism when

**Comment by kol1th** [comment]
Delete this fucking pr 🔫

**Comment by Orangerush** [comment]
del ur life

**Pull request merged**